easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -204,6 +206,13 @@ def configure_step(self):
         # always include -w to supress warnings
         dflags.append('-w')
 
+        if LooseVersion(self.version) >= LooseVersion("6.6"):
+            dflags.append(" -Duse_beef")
```

Comment:
Shouldn't this be below the "if libbeef" stmt? Or does it build an internal version if not found externally?

Reply:
yes, by default it builds the internal version